compose/web/src/jsMain/kotlin/androidx/compose/web/attributes/Attrs.kt

```diff
@@ -336,3 +346,10 @@ private val setInputValue: (HTMLInputElement, String) -> Unit = { e, v ->
 /* Img attributes */
 fun AttrsBuilder<Tag.Label>.forId(value: String?): AttrsBuilder<Tag.Label> =
     attr("for", value)
+
+/* Table attributes */
+fun AttrsBuilder<Tag.Th>.scope(value: String?): AttrsBuilder<Tag.Th> =
```

Comment:
What do you think about adding some more specific type for the scope value? It could be an enum for example: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/th#attr-scope

We have some enums and sealed classes in PredefinedAttrValues.kt. You can add it there. You can see some sealed classes there, although in this case enum seems to be sufficient.

Reply:
Yes, it was intentional to support exactly this kind of calling.